docs/cron-workflows.md

```diff
@@ -2,10 +2,95 @@
+|           `suspend`          |         `false`        | If `true` Workflow scheduling will not occur. Can be set from the CLI, GitOps, or directly                                                                                                                                             |
+|      `concurrencyPolicy`     |         `Allow`        | Policy that determines what to do if multiple `Workflows` are scheduled at the same time. Available options: `Allow`: allow all, `Replace`: remove all old before scheduling a new, `Forbid`: do not allow any new while there are old |
+| `startingDeadlineSeconds`    |           `0`          | Number of seconds after the last successful run during which a missed `Workflow` will be run                                                                                                                                           |
+| `successfulJobsHistoryLimit` |           `3`          | Number of successful `Workflows` that will be persisted at a time                                                                                                                                                                      |
+| `failedJobsHistoryLimit`     | `1`                    | Number of failed `Workflows` that will be persisted at a time                                                                                                                                                                          |
+
+## Managing `CronWorkflow`
+
+### CLI
+
+`CronWorkflow` can be created from the CLI by using basic commands:
+
+```sh
+$ argo cron create cron.yaml
+Name:                          test-cron-wf
+Namespace:                     argo
+Created:                       Mon Nov 18 10:17:06 -0800 (now)
+Schedule:                      * * * * *
+Suspended:                     false
+StartingDeadlineSeconds:       0
+ConcurrencyPolicy:             Forbid
+
+$ argo cron list
+NAME           AGE   LAST RUN   SCHEDULE    SUSPENDED
+test-cron-wf   49s   N/A        * * * * *   false
+
+# some time passes
+
+$ argo cron list
+NAME           AGE   LAST RUN   SCHEDULE    SUSPENDED
+test-cron-wf   56s   2s         * * * * *   false
+
+$ argo cron get test-cron-wf
+Name:                          test-cron-wf
+Namespace:                     argo
+Created:                       Mon Nov 18 10:17:06 -0800 (4 minutes ago)
+Schedule:                      * * * * *
+Suspended:                     false
+StartingDeadlineSeconds:       0
+ConcurrencyPolicy:             Replace
+LastScheduledTime:             Mon Nov 18 10:21:00 -0800 (51 seconds ago)
+Active Workflows:              test-cron-wf-rt4nf
+```
+
+### GitOps via Argo CD
+
+`CronWorkflow` resources can be managed with GitOps by using [Argo CD](https://github.com/argoproj/argo-cd)
```

Comment:
Shout out 🐙

Reply:
this seems tangential to the docs, do we really need it?

Reply:
Definitely a bit tangential, but I only recently realized this could be done thought it would be useful to have it documented. Can totally remove if necessary

Reply:
I don't feel strongly about this. Leave it if you like. Docs are living things.

Reply:
Let's leave it then